Uplift/Models/Facility.swift

```diff
@@ -33,6 +33,10 @@ struct Facility: Hashable {
     /// The status of this facility.
     let status: Status?
 
+    /// The equipments of this facility.
+
+    let equipment: [Equipment]
```

Comment:
Let's keep the properties in alphabetical order (besides `id`) for styling purposes.